packages/expo-random/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### 🛠 Breaking changes
 
+- Remove side-effect, binding the manifest environment variables to the app's `process.env` ([#11559](https://github.com/expo/expo/pull/11559) by [@EvanBacon](https://github.com/EvanBacon))
```

Comment:
This makes it sound like by removing the side effect, we now bind manifest env vars to process.env.